src/tests/specs/iteration.spec.ts

```diff
@@ -30,16 +30,16 @@ describe('Iteration test', () => {
     let dropdownIteration1 = 'Iteration_5',
       updateIteration = 'Iteration 0123',
       workItemTitle1 = 'Workitem_Title_10';
-    await planner.sidePanel.ready();
+    await planner.sidePanel.clickScenarios();
```

Comment:
Scenarios should be selected by default on loading the planner page. why do we explicitly need to click on it?

Reply:
@rgarg1 This is to reset the current state. We need to ensure the page is in its initial state before the next test starts.
@Raunak1203 Can we have a `resetState()` function in `afterEach` that would
1. Close quickpreview if it is open
2. Clear all filters
3. Click on Scenarios.

Reply:
yes we can include resetState() based on ifCondition

Reply:
:+1: Let's do that @Raunak1203